README.md

```diff
@@ -32,4 +32,6 @@ On `require`, it is indexed and loaded into memory. Finding postal codes is just
 ## Alternatives in other languages
 
 These community-maintained libraries are based on the same dataset and offer equivalent features in other languages:
-- [Dart](https://pub.dartlang.org/packages/code_postaux)
+
+- [Dart](https://pub.dartlang.org/packages/code_postaux) by [@Kleak](https://github.com/Kleak/code_postaux).
```

Comment:
Yeah… But I thought it would be easier to navigate to the person from the repo than the other way around, and other implementers might be curious of how you implemented it.
I'll add links to both.